File: api/src/main/java/com/hirrao/javaee/entity/Article.java
package com.hirrao.javaee.entity;


import com.baomidou.mybatisplus.annotation.TableName;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.data.annotation.Id;


@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@TableName("articles")
public class Article {
    @Id
    private Long id;
    private Long articleId;
    private String title;
    private String description;
    private String image;
    private String content;


    // getters and setters
}

File: api/src/main/kotlin/com/hirrao/javaee/entity/Article.kt
package com.hirrao.javaee.entity

import com.baomidou.mybatisplus.annotation.TableName
import org.springframework.data.annotation.Id


@TableName("articles")
data class Article(
    @Id var id: Long? = null,

    var title: String? = null,
    var description: String? = null,
    var image: String? = null,
    var content: String? = null // getters and setters,
)